src/ui/pages/config/WidgetGrouping.js

```diff
@@ -24,13 +25,19 @@ const WidgetGrouping = ({
       <Collapse in={opened}>
         <div>
           <div className={itemAreaClassNames.join(' ')}>
-            {widgets.map(widget => (
-              <DraggableContentWidgetElement
+            {widgets.map(widget => (view === 'grid' ? (
```

Comment:
please use constants for view types (grid or list)

Reply:
done ✅